packages/components/src/text-control/index.styles.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { inputPlaceholder, inputOutline, focussedInputOutline, paddingX, paddingY } from '../_styles';
+
+export const Input = styled.input`
+	box-sizing: border-box;
+	width: 100%;
+	${ paddingX( 2 ) }
+	${ paddingY( 1 ) }
+	/* text({variant: 'body'}) */
```

Comment:
Double-checking! Is this a comment necessary for now?

Reply:
Yeah, its still waiting for https://github.com/WordPress/gutenberg/pull/18495